OpenEdXMobile/src/main/java/org/edx/mobile/view/app_nav/CourseUnitNavigationActivity.kt

```diff
@@ -653,7 +653,7 @@ class CourseUnitNavigationActivity : BaseFragmentActivity(), CourseUnitFragment.
 
     override fun showGoogleCastButton(): Boolean {
         val component = getCurrentComponent()
-        return if (component is VideoBlockModel) {
+        return if (config.isChromeCastEnabled && component is VideoBlockModel) {
```

Comment:
I think there is no need to check the config here, cuz before calling the `showGoogleCastButton` `BaseAppActivity` has the null check at [Line#93](https://github.com/openedx/edx-app-android/pull/1822/files#diff-956469254164e43988f9eaaba92a2b5ca2c10326f6c9229243750fb0963e9313R93)

Reply:
Add this check here because this method is calling from multiple places.